internal/controller/http/v1/devicemanagement.go

```diff
@@ -43,6 +43,9 @@ func NewAmtRoutes(handler *gin.RouterGroup, d devices.Feature, amt amtexplorer.F
 		h.POST("power/bootOptions/:guid", r.setBootOptions)
 		h.POST("power/bootoptions/:guid", r.setBootOptions)
 		h.GET("power/capabilities/:guid", r.getPowerCapabilities)
+		h.GET("power/bootService/:guid", r.getBootService)
+		h.GET("power/bootSourceSetting/:guid", r.getBootSourceSetting)
```

Comment:
we might want to put some thought into these endpoints. At first glance, power doesn't seem to pass the vibe check. Something like `boot/source/:guid` seems way more intuitive to me, and `boot/source/settings/:guid` or `boot/source/:guid/settings`.

Reply:
removed these end points